### ISSUE TITLE
Docs: Bring back the acknowledgements page

### ISSUE THREAD

**Issue by monkeyiq** [filesender/filesender#131]
This is likely to be a two page solution, with a specific acknowledgements page and citation of facilities on a per release basis.

**Comment by WebSpider** [comment]
We could introduce a CHANGELOG, and use this as a basis.

**Comment by meijer** [comment]
And acknowledge committers on a per-release basis in that CHANGELOG? It would be a start. Any idea whether you could update that CHANGELOG automatically?

**Comment by WebSpider** [comment]
There are indeed sufficient examples where github actions generates a CHANGELOG.
I'll work on a PR, unless someone objects.

**Comment by WebSpider** [comment]
I've done some short investigations on the CHANGELOG to accomodate some form of acknowledgements.
Investigated projects are:

- https://github.com/conventional-changelog/conventional-changelog
- https://github.com/github-changelog-generator/github-changelog-generator

Both seem to have similar possibilties in generating a changelog, the second is suited only for github hosted projects (and integrates with MR's, issues, etc)

To do give this comparison some body, there is the generated changelog from these projects. I've done no tweaking other than what I needed to do to get it to run:

Conventional-changelog with all default options: https://github.com/WebSpider/filesender/blob/changelogs/conventional-changelog/default/CHANGELOG.md
Conventional-changelog with angularjs commit styles: https://github.com/WebSpider/filesender/blob/changelogs/conventional-changelog%2Fangular/CHANGELOG.md
Conventional-changelog with eslint commit styles: https://github.com/WebSpider/filesender/blob/changelogs/conventional-changelog%2Feslint/CHANGELOG.md
github_changelog_generator filtered to exclude all releases prior to filesender-2 and all svn branches: https://github.com/WebSpider/filesender/blob/changelogs/github_changelog_generator%2Fsince-filesender2.0/CHANGELOG.md

My personal preference is using the github_changelog_generator, and accepting that it's integration works well with GitHub only. If we decide to move away from GitHub, I don't see a problem in storing the CHANGELOG in some kind of archive mode, and starting a new CHANGELOG at that point.

**Comment by meijer** [comment]
Thanks for the effort! I checked the various output and agree with you: if these are the options then the github_changelog_generator looks best. There's always that issue when using an integrated platform like Github, the ease of use (=less time spent on overhead) and risk of having to invest time when you need to move platform. The output is .md, so the risk is in auto-generating a changelog. Acceptable imho. 

Am I right in guessing that github_changelog_generator could run as part of the standard process to generate a release?

**Comment by WebSpider** [comment]
You are right. Running this can either be done from commandline for just generating the file, or as part of CI.

An example of the latter is a GitHub Action that generates a release for you, with many options to make it just the way you want:
https://github.com/marketplace/actions/release-changelog-builder
